vuepress/docs/next/tutorials/create/ms/run-local.md

```diff
@@ -2,139 +2,77 @@
+## Start the Micro Frontends
+### Run the conference-table MFE
+1. From another shell, start the conference-table micro frontend from the project root directory. This command runs React in development mode so any changes you make to the source files should be immediately seen in the browser. 
 ``` sh
-  docker-compose -f src/main/docker/keycloak.yml up
+ent bundle run conference-table
 ```
+If you are not logged in, you'll be redirected to do so. Log in using the following credentials. 
+* Username: user 
+* Password: user\
+Once logged in, you will see the table widget with some generated data.
 
->Note: If you need to install 
-Docker Compose, follow this guide:
-<https://docs.docker.com/compose/install/>
-
-
-### Start the Microservice
-
-1.  Start the generated microservice with the command:
-
-        ./mvnw
-
-#### Notes
-1. If you want to reset the widget data (for example, if you deleted all rows from the table widget), and you selected "H2 with disk-based persistence" during microservice generation, you can delete the target folder, restart the microservice, and the data will be regenerated.
-2. The `service-url` variable is the microservice API URL.
-
-### Start the Table Widget
-
-1.  Go to the table widget folder in your project:
-
-        cd ui/widgets/YOUR-ENTITY-NAME/tableWidget
-
-2.  Install and start your widget with the command:
-
-        npm install && npm start
-
-3.  When the widget starts, a browser window opens the widget URL
-
-4.  If you’re not logged in, you'll be redirected to the login page.
-
-5.  Log in using:
-
-        Username: user
-        Password: user
-
-6.  Once logged in, you can see the table widget with generated data.
-
-### Start the Form Widget
-
-1.  If you are already running a widget, click `Ctrl+C` to end the process
-
-2.  Go to the form widget folder in your project:
-
-        cd ui/widgets/YOUR-ENTITY-NAME/formWidget
-
-3.  Install and start your widget with the command:
-
-        npm install && npm start
-
-4.  When the widget starts, a browser window opens the widget URL 
-
-5.  If you’re not logged in, you'll be redirected to the login page
-
-6.  Log in using:
 
-        Username: user
-        Password: user
+### Run the conference-form MFE
 
-7.  Once logged in, you will see the widget form.
+1.  If you are still running the previous micro frontend, click `Ctrl+C` to end the process
 
-#### Form Widget notes:
+2. Start up the conference-form MFE 
+``` sh
+ent bundle run conference-form
+```
+The form to enter the name and location for conferences shoud open in your browser. You may enter and save new data to see it in the other MFEs.
 
-If you want to modify a different row in the database, edit the `index.html` file in the public folder `ui/widgets/YOUR-ENTITY-NAME/detailsWidget/public`. Change the `id` attribute in this line:
+>If you want to modify a different row in the database, edit `microfrontends/conference-form/public/index.html` file. Change the `id` attribute in this line:
```

Comment:
@nshaw, cannot find the id attribute in the index.html

Reply:
@jyunmitch yeah, because I probably broke it in my conversion. Let's check that with the new blueprint